crates/ra_ide/src/syntax_highlighting/tests.rs

```diff
@@ -291,6 +291,9 @@ fn main() {
 fn test_highlight_doctest() {
     check_highlighting(
         r#"
+/// ```
```

Comment:
```suggestion
        r#"/// ```
```

Otherwise it starts with a newline.

Reply:
@bjorn3 the original code started with a newline, and even with the newline it triggered the bug. I think it's worth it to keep it as is so the test is slightly more readable. If you feel strongly for making the change, I can go ahead and remove the newline.

Reply:
I didn't know if the bug reproduced with a newline at the top. You can keep the newline.

Reply:
Thanks for the fast response :)